pocs/observatory.py

```diff
@@ -107,6 +128,42 @@ def current_observation(self, new_observation):
     def has_dome(self):
         return self.dome is not None
 
+
+##########################################################################
+# Device Getters/Setters
+##########################################################################
+
+    def add_camera(self, cam_name, camera):
```

Comment:
Do you intend these two add/remove methods to be used to dynamically add and remove cameras during operation?

Reply:
Yes, eventually that would be the goal.